numba_cuda/numba/cuda/stubs.py

```diff
@@ -1,13 +1,12 @@
 """
-This scripts specifies all PTX special objects.
+This script specifies all PTX special objects.
 """
 import numpy as np
 from collections import defaultdict
 import functools
 import itertools
 from inspect import Signature, Parameter
 
-
```

Comment:
There should be two blank lines between class / function definitions and other code - could you please add back the blank line?